src/constants.rs

```diff
@@ -254,7 +254,9 @@ fn test_zs_and_us<C: CurveAffine>(base: C, z: &[u64], u: &[[[u8; 32]; H]], num_w
     for ((u, z), window_points) in u.iter().zip(z.iter()).zip(window_table) {
         for (u, point) in u.iter().zip(window_points.iter()) {
             let y = *point.coordinates().unwrap().y();
-            let u = C::Base::from_bytes(u).unwrap();
+            let mut u_repr = <C::Base as PrimeField>::Repr::default();
+            u_repr.as_mut().copy_from_slice(u);
+            let u = C::Base::from_repr(u_repr).unwrap();
```

Comment:
Since `test_zs_and_us` is only used where `z`s, `u`s are `pallas::Base` elements, let's just use the concrete type here.
```suggestion
            let u = pallas::Base::from_repr(u).unwrap();
```